pkg/skaffold/build/gcb/desc_test.go

```diff
@@ -71,3 +71,35 @@ func TestBuildDescription(t *testing.T) {
 
 	testutil.CheckDeepEqual(t, expected, *desc)
 }
+
+func TestPullCacheFrom(t *testing.T) {
+	artifact := &v1alpha2.Artifact{
+		ImageName: "nginx",
+		ArtifactType: v1alpha2.ArtifactType{
+			DockerArtifact: &v1alpha2.DockerArtifact{
+				DockerfilePath: "Dockerfile",
+				CacheFrom:      []string{"from/image1", "from/image2"},
+			},
+		},
+	}
+
+	builder := Builder{
+		GoogleCloudBuild: &v1alpha2.GoogleCloudBuild{
+			DockerImage: "docker/docker",
+		},
+	}
+	desc := builder.buildDescription(artifact, "bucket", "object")
+
+	expected := []*cloudbuild.BuildStep{{
+		Name: "docker/docker",
```

Comment:
This is unrelated to your change, but should we be using https://github.com/GoogleCloudPlatform/cloud-builders/tree/master/docker instead of docker/docker as the image?

Reply:
By default, we do. This is just a test

Reply:
Ah, great.